### ISSUE TITLE
Select-Object: Allow combining -Skip and -SkipLast

### ISSUE THREAD

**Issue by mklement0** [PowerShell/PowerShell#11752]
See also: #13769

# Summary of the new feature/enhancement

`Select-Object`'s `-Skip` and `-SkipLast` parameters are currently in separate parameter sets, which syntactically prevents using them together.

However, there seems to be no good _conceptual_ reason to prevent that, as it is conceivable that you may want to skip elements at _both_ the beginning and end:

```powershell
# WISHFUL THINKING
1..5 | Select-Object -Skip 1 -SkipLast 1
2
3
4
```

* `-Skip`  should be applied first.

* `-SkipLast` must remain mutually exclusive with `-First` and `-Last`(which themselves can be combined with `-Skip`).

If the suggestion in #11750 is implemented, the same combination of parameters should also be allowed with `Get-Content`.

**Comment by scotthardwick** [comment]
Great idea, +1

**Comment by mklement0** [comment]
Glad to hear it, @scotthardwick - if this is green-lighted, it should also be implemented for `Get-Content`, if #11750 gets green-lighted too.

On a quick meta note (I hope that's OK - you've indicated that you've only recently started participating here): 

* To show _support_ for a proposal _without further qualification_, it's better to just do a thumbs-up on the original post, to keep the "signal-to-noise" ratio down (comments exchanges can get very lengthy, making it hard to come back to an issue later and grasp its gist).

* By contrast, to voice *opposition* it's better to not only cast a thumbs-down vote, but to complement that with a comment as to _why_.

GitHub now offers a way to _hide_ one's previous comments after the fact (which is usually preferable to outright _deletion_, so as not to rewrite history), with the ability to choose from a variety of reasons for hiding.

**Comment by SeeminglyScience** [comment]
The logic is already there to handle this, the parameter `Skip` just needs to be decorated as included in the parameter set.

https://github.com/PowerShell/PowerShell/blob/788ffde65c96dbb0f815faebbc60fa9901ed52c8/src/Microsoft.PowerShell.Commands.Utility/commands/utility/Select-Object.cs#L149-L155

```diff
        /// <summary>
        /// Skips the specified number of items from top when used with First, from end when used with Last.
        /// </summary>
        /// <value></value>
        [Parameter(ParameterSetName = "DefaultParameter")]
+       [Parameter(ParameterSetName = "SkipLast")]
        [ValidateRange(0, int.MaxValue)]
        public int Skip { get; set; }
```
